templates/donors.hbs

```diff
@@ -0,0 +1,18 @@
+<div class="donors">
+  <div class="donors_image">
+    {{{asset image}}}
+  </div>
+  <div class="donors_info">
+    <p class="donors_description">{{description}}</p>
+    <div class="donors_link_container">
+      <div class="donors_ctaPrimary_link" aria-label="Button for donations">
+        {{{module ctaPrimary}}}
+      </div>
+      <div class="donors_ctaSecondary_link" aria-label="Button why donate">
```

Comment:
Same validation but for `ctaSecondary`
{{#if ctaSecondary}}
  <div class="donors_ctaSecondary_link" aria-label="Button why donate">
    {{{module ctaPrimary}}}
  </div>
{{/if}}